src/google/protobuf/compiler/subprocess.cc

```diff
@@ -258,7 +258,7 @@ bool Subprocess::Communicate(const Message& input, Message* output,
 }
 
 string Subprocess::Win32ErrorMessage(DWORD error_code) {
-  char* message;
+  TCHAR* message = nullptr;
```

Comment:
Instead of using TCHAR (and wstring), can you change the code to use the ASCII version of all win API functions? For example, instead of FormatMessage, use FormatMessageA.